### PR TITLE
Add godoc to the Commit function as it is exported

### PR DESCRIPTION
**Pull request by maramihali**

### CODE REVIEW

File: safehttp/plugins/cors/cors.go [hunk 1]
```diff
@@ -154,6 +154,7 @@ func (it *Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRe
 	return safehttp.Result{}
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/cors/cors.go:157]: nit: the comma actually changes the meaning of the sentence.

Comment (kele) [inline safehttp/plugins/cors/cors.go:157]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

File: safehttp/plugins/csp/csp.go [hunk 2]
```diff
@@ -215,6 +215,7 @@ func (it Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingReq
 	return safehttp.Result{}
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/csp/csp.go:218]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

File: safehttp/plugins/fetch_metadata/fetch_metadata.go [hunk 3]
```diff
@@ -169,6 +169,7 @@ func (p *Plugin) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRequest,
 	return safehttp.Result{}
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/fetch_metadata/fetch_metadata.go:172]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

File: safehttp/plugins/hsts/hsts.go [hunk 4]
```diff
@@ -97,6 +97,7 @@ func (it Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingReq
 	return safehttp.NotWritten()
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/hsts/hsts.go:100]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

File: safehttp/plugins/staticheaders/staticheaders.go [hunk 5]
```diff
@@ -38,6 +38,7 @@ func (Plugin) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingRequest, cf
 	return safehttp.NotWritten()
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/staticheaders/staticheaders.go:41]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

File: safehttp/plugins/xsrf/xsrf.go [hunk 6]
```diff
@@ -138,6 +138,7 @@ func (i *Interceptor) Before(w *safehttp.ResponseWriter, r *safehttp.IncomingReq
 	return safehttp.NotWritten()
 }
 
+// Commit is a no-op required to satisfy the safehttp.Interceptor interface.
```
Comment (kele) [inline safehttp/plugins/xsrf/xsrf.go:141]:
```suggestion
// Commit is a no-op, required to satisfy the safehttp.Interceptor interface.
```

**Review by kele — CHANGES_REQUESTED** [review]

**Review by kele — APPROVED** [review]

File: safehttp/plugins/cors/cors.go (hunk 1 above)
Comment (maramihali) [reply safehttp/plugins/cors/cors.go]: Right, my bad

**Pull request merged**